docs/eventing/migration/README.md

```diff
@@ -0,0 +1,2 @@
+Knative Eventing in version 0.13 does deprecate a few resources. This document provides a collection of migration examples:
```

Comment:
Is PingSource the only thing deprecated for 0.13?
If so, we should instead just make this a section for "Migrating CronJobs to PingSource" under Eventing - it will be easier for readers to find what they're looking for. We can always expand things later if there are other migrations required.

WDYT @mattmoor @carieshmarie ?